### ISSUE TITLE
Heirlooms Kept (Rarity/Name)

### ISSUE THREAD

**Issue by Fofalus** [zininzinin/AutoTrimps#176]
It would be nice if it kept heirlooms you had named and always kept higher rarity in storage. I have been letting it do many fast runs to 125 and am never sure if I lost out on rare heirlooms.

**Comment by genbtc** [comment]
I have invented a "Protect Heirloom" button that will prevent the Auto Heirloom from recycling it for some reason. This will be coming soon. If you did not realize, The "Auto Heirlooms" function assesses which heirlooms are best, based on the rarity and mods and sends the rest to be recycled on portal.

**Comment by Fofalus** [comment]
I saw your fork of this and have moved to that with its protect function. I don't believe that it always keeps based on rarity because I have caught it throwing away legendary items to keep epic items in their place. If I am understanding correctly it should always keep legendary items above epic. This may be working correctly in your fork and not the original version I was using I am unsure.

**Comment by genbtc** [comment]
no, they are identical. Rarity takes precedence above mods, but the mods CAN outweigh rarity, and in that case, it would be such a great outweighal that you should listen to it that, it was indeed a sheit legendary.

**Comment by Fofalus** [comment]
Ah the tooltip says Rarity trumps all stat evaluation but I guess that only applies to carrying. It was a very garbage one for normal purposes but one I had kept for scientist runs.

**Comment by genbtc** [comment]
yes now im confused. It does say it trumps all stat evaluations, so how you think it threw the legendary away I do not know. I will look into it.

**Comment by Fofalus** [comment]
I still have the very garbage shield and an equally terrible magnificent staff. I will let it continue running with them unprotected to see if I can reproduce it.

**Comment by KMStanton** [comment]
it doesb't keep the best heirloom by rarity, what's wrong?
